website/client/src/components/groups/groupFormModal.vue

```diff
@@ -254,7 +255,8 @@ label.custom-control-label(v-once) {{ $t('allowGuildInvitationsFromNonMembers')
         <button
           v-if="workingGroup.id"
           class="btn btn-primary btn-md"
-          :disabled="!workingGroup.name || !workingGroup.description"
+          :disabled="(isParty && !workingGroup.name) ||
```

Comment:
I think this can be simplified to `!workingGroup.name || (!isParty && !workingGroup.description)`

Reply:
Done, thanks!